website/docs/reference/resource-configs/store_failures.md

```diff
@@ -110,3 +108,20 @@ tests:
 </TabItem>
 
 </Tabs>
+
+## FAQs
+
+<detailsToggle alt_header="Receiving a 'permissions denied for schema' error">
+
+If you're receiving a `Adapter name adapter: Adapter_name error: permission denied for schema dev_usernamea_dbt_test__audit`, this is most likely due to your user not having permission to create new schemas, despite having owner access to your own development schema.
+
+To resolve this, you need proper authorization to create or access custom schemas. Run the following SQL command in the data platform environment:
+
```

Comment:
Hiya @mirnawong1 

(non-blocker suggestion) would it be worthwhile adding an image to show users where they can add these credentials?

Kind Regards
Natalie

Reply:
thats a great question and i did think about this but i think the user has to do this in their own data platform settings. I'm going to loop @wusanny to confirm that though

Reply:
Thank you @mirnawong1 for writing these up. The authorization needs to be done in their respective data platform, that is right. 

Hopefully am responding to these correctly, just couple things:
- tiny typo on "dev_usernamea_dbt_test__audit" in line 116 - there is an extra 'a'
- the exact authorization query may differ from one data platform to another, not sure if it's something worth mentioning?

Reply:
brilliant, thanks so much @wusanny ! will update and thanks for those catches! 🦅👀

Reply:
folded in those changes, thank you again!